### PR TITLE
snapshots

### PR DESCRIPTION
**Pull request by bsach64**

### CODE REVIEW

**Comment by bsach64** [comment]
abhi ke liye same dir me rehne dete hai

File: client/client.go [hunk 1]
```diff
@@ -82,3 +92,28 @@ func uploadChunks(sftpClient *sftp.Client, chunks map[string][]byte) error {
...
+		return err
+	}
+	defer remoteFile.Close()
+
+	for key := range chunks {
+		chunkFileName := fmt.Sprintf("%s.chunk\n", key)
+		if _, err := remoteFile.Write([]byte(chunkFileName)); err != nil {
+			return err
+		}
+	}
```
Comment (Unic-X) [inline client/client.go:115]: how can we reconstruct? 
We need to know the order in which a file is chunked.

Comment (Unic-X) [reply client/client.go:115]: Instead, we can maybe map each number to hash 
reconstruction_id = hash of chunk?

Comment (bsach64) [reply client/client.go:120]: add kar diya hu reconstruction

**Comment by Unic-X** [comment]
Reconstruction ke steps bata @bsach64

**Comment by bsach64** [comment]
> Reconstruction ke steps bata @bsach64 


./goback reconstruct

**Review by Unic-X — APPROVED** [review]
Sahi hai krdeta merge

**Pull request merged**